client/src/helpers/APICalls/request.ts

```diff
@@ -0,0 +1,21 @@
+import { FetchOptions } from '../../interface/FetchOptions';
+import RequestData from '../../interface/Request';
+
+const fetchRequests = async (): Promise<RequestData> => {
```

Comment:
This code looks like it belongs to the Bookings:IN ticket, I will review it in the relevant PR :)

Reply:
See you there!